skipruntime-ts/ts/src/internals/skipruntime_impl.ts

```diff
@@ -255,8 +255,7 @@ export class SKStoreImpl extends SkFrozen implements SKStore {
     const name = computeObj.constructor.name;
     const lazyHdl = this.context.lazy(
       name,
-      (selfHdl: LazyCollection<K, V>, key: K) =>
-        computeObj.compute(selfHdl, key),
+      computeObj.compute.bind(computeObj),
```

Comment:
Open question: do we generally prefer `obj.method.bind(obj)` over `(arg1, arg2) => obj.method(arg1, arg2)`?

Reply:
With `(arg1, arg2) => obj.method(arg1, arg2)`, I want to simplify to just `obj.method` and then it fails... so `obj.method.bind(obj)`, although ugly, makes it explicit

Reply:
In that case let's open an issue with a reminder to use that style across the board (there are instances of the lambda variant in the skipruntime client or server).